test/e2e/fixtures.go

```diff
@@ -109,6 +108,17 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error) {
 	return data, nil
 }
 
+func createDirForCoverage(covDir string) error {
+	if err := os.Mkdir(covDir, 0700); err != nil {
+		return fmt.Errorf("Error when creating coverage directory '%s': %v", covDir, err)
+	}
+	err := os.Chmod(covDir, 0777)
+	if err != nil {
+		return fmt.Errorf("Failed to granting permission to %s: %v", covDir, err)
+	}
```

Comment:
Please explain why you use `Chmod` after `Mkdir` instead of using `os.Mkdir(covDir, 0777)`?

Also please add a comment to explain why these permissions (read / write for all users) are required